### PR TITLE
Updated Professor Card

### PR DESCRIPTION
**Pull request by Jake3231**
## Overview

This PR implements the updated professor card design.

(@AdiC03 new PR to change base and compare branches)

See #126

### CODE REVIEW

File: components/common/ProfessorCard/ProfessorCard.tsx [hunk 1]
```diff
@@ -9,90 +13,100 @@ type ProfessorCardProps = {
   averageDifficulty: number;
   takingAgain: number;
   numRatings: number;
+  //numEvaluation: number; (Need to include percentage for evalution)
```
Comment (Jake3231) [inline components/common/ProfessorCard/ProfessorCard.tsx:16]: @AdiC03 Is this expected?

File: components/common/ProfessorCard/ProfessorCard.tsx [hunk 2]
```diff
@@ -1,3 +1,7 @@
+// Only problem is there's a black sillhoute that appears
+// Because of the smaller professor card. Most likely due to the css element.
```
Comment (Jake3231) [inline components/common/ProfessorCard/ProfessorCard.tsx:2]: This should be resolved before merging if possible.

**Review by Jake3231 — COMMENTED** [review]
Left a few questions @AdiC03

**Comment by Jake3231** [comment]
@AdiC03 Do you expect to be able to make changes to this PR ahead of merging?

**Comment by AdiC03** [comment]
> @AdiC03 Do you expect to be able to make changes to this PR ahead of merging?

Not really. There is a small problem regarding the background css but I can't really seem to figure it out (very minor issue)

**Comment by Jake3231** [comment]
> Not really. There is a small problem regarding the background css but I can't really seem to figure it out (very minor issue)

Have you asked around in #trends-engineering about addressing that?

**Comment by Jake3231** [comment]
@TyHil What do you suggest for this PR?

**Comment by TyHil** [comment]
Looks like the "Based on x ratings" link opens to a search for the professor in any school, not just UTD.
Also the "Would Take Again" box shows as a rectangle for me, we should fix that to a square like in the Figma.

Other than that, this looks phenomenal!

File: components/common/ProfessorCard/ProfessorCard.tsx (hunk 2 above)
Comment (TyHil) [reply components/common/ProfessorCard/ProfessorCard.tsx:2]: I think this ought to be fixed when we put all the components together as this isn't really a problem with the professor card but just a side effect of displaying it in the current UI.

**Review by TyHil — CHANGES_REQUESTED** [review]
Looks like the "Based on x ratings" link opens to a search for the professor in any school, not just UTD.
Also the "Would Take Again" box shows as a rectangle for me, we should fix that to a square like in the Figma.

Other than that, this looks phenomenal!

**Comment by Jake3231** [comment]
Is this still relevant @iamwood ?

**Pull request closed without merging**